docs/src/pages/it/getting-started.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Come iniziare
+description: Una guida introduttiva ad Astro.
+---
+
+Astro è un generatore moderno di siti statici. Scopri di più su Astro dalla [nostra homepage](https://astro.build/) o dal nostro [post di rilascio](https://astro.build/blog/introducing-astro). Questa pagina è una panoramica della documentazione di Astro e di tutte le risorse correlate.
+
+## Prova Astro
+
+Il modo più semplice per provare Astro è lanciare `npm init astro` in una nuova cartella nella tua macchina. La procedura guidata da CLI ti assisterà nell'avvio di un nuovo progetto Astro.
+
+Puoi iniziare da subito con Astro in 5 passi semplici, attraverso la nostra [Guida Rapida all'Avvio](/quick-start).
+
+In alternativa, leggi la nostra [Guida all'Installazione](/installation) per una panoramica più completa su come impostare Astro.
+
+### Esempi
+
+Se preferisci imparare Astro tramite esempi, dai un'occhiata alla nostra [libreria completa degli esempi](https://github.com/snowpackjs/astro/tree/main/examples) su GitHub.
+
+Puoi testare uno qualsiasi di questi esempi sulla tua macchina locale lanciando `npm init astro` con la flag `--template` da CLI. La flag `--template` supporta anche template di terze parti della community.
+
+```bash
+# Avvia la procedura guidata e usa questo template ufficiale
+npm init astro -- --template [OFFICIAL_EXAMPLE_NAME]
```

Comment:
Should the placeholders here and below be translated? Something like

```suggestion
npm init astro -- --template [NOME_ESEMPIO_UFFICIALE]
```

Reply:
Good catch, thanks!